electron/ironfish/AccountManager.ts

```diff
@@ -24,11 +24,12 @@ class AccountManager implements IIronfishAccountManager {
     const accounts = this.node.wallet.listAccounts()
 
     const result: CutAccount[] = await Promise.all(
-      accounts.map(async account => ({
+      accounts.map(async (account, index) => ({
         id: account.id,
         name: account.name,
         publicAddress: account.publicAddress,
         balance: await this.balance(account.id),
+        order: index,
```

Comment:
suggestion: Instead of adding order as a property of accounts, could we have a utility function that return an account's order based on its id? That seems easier than maintaining this property throughout the application.